model_zoo/spleen_ct_segmentation_pytorch/README.md

```diff
@@ -0,0 +1,61 @@
+# Description
+Load pre-trained model from MONAI model-zoo with pytorch and train it on a new dataset.
```

Comment:
I would suggest to change the description to:
"Load the pre-trained weights from a MONAI model-zoo model and execute the transfer-learning in a typical PyTroch training program based on it."
What do you think?

Thanks.